usrsctplib/netinet/sctp_pcb.c

```diff
@@ -5325,7 +5325,7 @@ sctp_add_vtag_to_timewait(uint32_t tag, uint32_t time, uint16_t lport, uint16_t
 				twait_block->vtag_block[i].rport = 0;
 				if (set == 0) {
 					/* Reuse it for my new tag */
-					twait_block->vtag_block[i].tv_sec_at_expire = now.tv_sec + time;
+					twait_block->vtag_block[i].tv_sec_at_expire = (uint32_t)now.tv_sec + time;
```

Comment:
Should be fixed in https://github.com/sctplab/usrsctp/commit/4ec13f704f974d5cdf1b98bff84c8918552b3115.